buildconfig/Jenkins/Conda/nightly_build_and_deploy.jenkinsfile

```diff
@@ -74,7 +82,9 @@ pipeline {
                 GITHUB_ACCESS_TOKEN = credentials("${GITHUB_TOKEN_CREDENTIAL_ID}")
             }
             steps {
-                sh '$WORKSPACE/buildconfig/Jenkins/Conda/update-conda-recipes.sh $GITHUB_ACCESS_TOKEN $GITHUB_USER_NAME'
+                catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
```

Comment:
Whilst this stage will show as failed, it will still run the next part of the pipeline anyway.

Reply:
https://stackoverflow.com/a/56986651

Reply:
Thanks. I see the idea that you might not want the whole pipeline to fail in some cases.
In this case if the update fails, which could be for several reasons, do we really want to continue?

Reply:
It covers us rebuilding, and nothing being different if a conda-build falls over for whatever reason. If there is nothing to update (e.g. release-next hasn't had a new push since last time) it won't pass, and the conda-builds will fall over. This came about because Windows conda-build failed, but every time it ran back through it gets to this stage and then fails, stopping another conda-build of the 3 platforms.